ApplicationLibCode/FileInterface/RifReaderInterface.cpp

```diff
@@ -151,10 +152,14 @@ void RifReaderInterface::importFaults( const QStringList& fileSet, cvf::Collecti
     }
     else
     {
+        bool isDataFileFound = false;
+
         foreach ( QString fname, fileSet )
```

Comment:
Avoid foreach macro. `for ( const QString& fname : fileSet)` should work

Reply:
fixed, we should replace all use of foreach